api/v1/invitations/InvitationController.php

```diff
@@ -243,4 +260,29 @@ public function decline(Request $illuminateRequest): JsonResponse
     {
         return $this->selectedHandler->decline($illuminateRequest);
     }
+
+    public function getMany(Request $illuminateRequest): JsonResponse
+    {
+        $context = $illuminateRequest->attributes->get('context'); /** @var \PKP\context\Context $context */
```

Comment:
@ipula  I do not think, we need these comments.

Reply:
this is from Dimithri's commit